C/28.c

```diff
@@ -3,21 +3,22 @@ int main() {
     int n, i, k=0;
     printf("Enter a positive integer: ");
     scanf("%d", &n);
+    if (n==1) {
+        printf("1 is neither prime nor composite.");
+    }
```

Comment:
Add an `exit(0)` inside `n==1` condition after the printf statement otherwise, the program will run and print 2 outputs.
Don't forget to add the resp. library for `exit(0)`